Polish documentation for audience and client scopes (#38484)
keycloak/keycloak: docs/documentation/server_admin/topics/clients/proc-evaluating-client-scopes.adoc

This will also show you the value of the *scope* parameter. This parameter needs to be sent from the application to the {project_name} OpenID Connect authorization endpoint.

[NOTE]
====

> See in the link:{securing_apps_base_link}/javascript-adapter[{project_name} JavaScript adapter] section if your application uses the javascript adapter for how to send the *scope* parameter with the desired value.

Reviewer: ```suggestion
If your application uses the {securing_apps_base_link}/javascript-adapter[{project_name} JavaScript adapter], see its section to learn how to send the *scope* parameter with the desired value.
```

====

> You can also simulate how the access token or ID token or UserInfo response issued to this client look like for the particular selected user and for the particular value of the `audience` parameter. Just a note

Reviewer: ```suggestion
You can also simulate how the access token, ID token, or UserInfo response issued to this client looks for a particular selected user and for a specific value of the `audience` parameter. Note
```

> that the parameter `audience` is supported just for the token exchange grant at this moment. It is recommended to leave it empty for simulating any other grant.

Reviewer: ```suggestion
that the `audience` parameter is currently only supported for the token exchange grant. It is recommended to leave it empty when simulating any other grant.
```
